### PR TITLE
Check that the DOC_MLGS list in Makefile.doc is correct

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: Makefile.doc [hunk 1]
```diff
@@ -316,6 +316,13 @@ DOC_MLGS := \
 DOC_EDIT_MLGS := $(wildcard doc/tools/docgram/*.edit_mlg)
 DOC_RSTS := $(wildcard doc/sphinx/*/*.rst) $(wildcard doc/sphinx/*/*/*.rst)
 
+REAL_DOC_MLGS := $(wildcard */*.mlg plugins/*/*.mlg)
+ifneq ($(sort $(DOC_MLGS)),$(sort $(REAL_DOC_MLGS)))
+missing_mlgs := $(filter-out $(REAL_DOC_MLGS),$(DOC_MLGS))
+extra_mlgs := $(filter-out $(DOC_MLGS),$(REAL_DOC_MLGS))
+$(error mlg file list mismatch in Makefile.doc: $(if $(missing_mlgs),$(missing_mlgs) not found) $(if $(extra_mlgs),$(extra_mlgs) not listed))
+endif
```
Comment (jfehrle) [inline Makefile.doc:324]: Isn't it conventional to indent the conditional lines?

**Review by jfehrle — APPROVED** [review]
Looks good to me.

File: Makefile.doc (hunk 1 above)
Comment (SkySkimmer) [reply Makefile.doc:324]: let's not think about indentation in makefiles

**Review by Alizter — APPROVED** [review]
merging soon

**Comment by Zimmi48** [comment]
Unfortunately, will need a rebase after the merge of #15241.

**Comment by Alizter** [comment]
@coqbot merge now

**Pull request merged**